### PR TITLE
Doremon in Python

### PR DESCRIPTION
**Pull request by bhaveshgarg2000**
Doremon in Python

### CODE REVIEW

**Comment by bhaveshgarg2000** [comment]
Hii @Amsal1 I have created new pull request please merge it
Thank You

File: Programs/doremon.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def Doraemon():
     black_aankha()
 
 
-if __name__ == '__main__':
+if __name__ == '__main__git status':
```
Comment (Amsal1) [inline Programs/doremon.py:300]: @bhaveshgarg2000 this doesn't seem right to me.

**Review by Amsal1 — CHANGES_REQUESTED** [review]
Line 300 should be: `if __name__ == '__main__':`

File: Programs/doremon.py (hunk 1 above)
Comment (bhaveshgarg2000) [reply Programs/doremon.py:300]: Let me check onces & update on it

**Comment by bhaveshgarg2000** [comment]
Hii @Amsal1 I updated the changes 
Sorry for the Inconvenience

**Review by Amsal1 — APPROVED** [review]

**Comment by bhaveshgarg2000** [comment]
Please add Hacktoberfest tag

**Comment by bhaveshgarg2000** [comment]
Thanks @Amsal1 ☺😊

**Pull request merged**